### PR TITLE
fix(@desktop/wallet): Wrong decimal place calculation on bridging

### LINKED ISSUE #8660: Wrong decimal place calculation on bridging
image

Then when I enter an amount, it shows that i don't have enough token

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
fixes #8660

### What does the PR do

Fixes the decimals point places in the bridge simple routing view.
please note that the balance exceeded error is not addressed in this PR but will be check under https://github.com/status-im/status-desktop/issues/8665

### Affected areas

Wallet

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

image

### CODE REVIEW

File: ui/imports/shared/views/NetworksSimpleRoutingView.qml [hunk 1]
```diff
@@ -114,7 +116,8 @@ RowLayout {
                 rightPadding: 5
                 implicitWidth: 150
                 title: chainName
-                subTitle: root.weiToEth(balance)
+                subTitle: selectedAccount && selectedAccount!== undefined && selectedAsset!== undefined ?
```
Comment (caybro) [inline ui/imports/shared/views/NetworksSimpleRoutingView.qml:119]: This should read `typeof selectedAccount !== undefined`

**Review by alaibe — APPROVED** [review]

File: ui/imports/shared/views/NetworksSimpleRoutingView.qml (hunk 1 above)
Comment (caybro) [reply ui/imports/shared/views/NetworksSimpleRoutingView.qml:119]: In other places too

Comment (Khushboo-dev-cpp) [reply ui/imports/shared/views/NetworksSimpleRoutingView.qml:119]: selectedAccount!== undefined works too, is this better ? @caybro

Comment (caybro) [reply ui/imports/shared/views/NetworksSimpleRoutingView.qml:119]: Ok then, thanks for verifying

Comment (Khushboo-dev-cpp) [reply ui/imports/shared/views/NetworksSimpleRoutingView.qml:119]: approve please? :)

**Review by caybro — APPROVED** [review]
LGTM :)

**Pull request merged**